manual/core/load_balancing/README.md

```diff
@@ -212,6 +212,43 @@ You can use your own implementation by specifying its fully-qualified name in th
 
 Study the [LoadBalancingPolicy] interface and the default implementation for the low-level details.
 
+### Using multiple policies
+
+The load balancing policy can be overridden in [configuration profiles](../configuration/#profiles):
+
+```
+datastax-java-driver {
+  load-balancing-policy {
+    class = DefaultLoadBalancingPolicy
+  }
+  profiles {
+    custom-lbp {
+      load-balancing-policy {
+        class = CustomLoadBalancingPolicy
```

Comment:
One thing I found really nice about this was that if all I wanted to do was override a lbp setting and still use `DefaultLoadBalancingPolicy`, I didn't actually have to specify the class name, just the name of the setting I wanted to change.